src/qmapshack/poi/CPoiIconCategory.cpp

```diff
@@ -0,0 +1,13 @@
+#include "CPoiIconCategory.h"
```

Comment:
There is a full path missing.

Reply:
Please make sure you add a full path to all headers in the cpp files. This helps to find stuff.